docker/make-requirements.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+# This script is used to generate or update requirements/lock files for R and Python packages
+# Requires that the `pip-tools` python package is installed.
+# Before running, make sure that the renv.lock file and installed libaries are
+# consistent with renv::snapshot() or renv::restore()
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# Python package lists
+pip-compile --no-annotate --strip-extras --output-file=requirements.txt requirements.in
+pip-compile --no-annotate --strip-extras --output-file=requirements_anndata.txt requirements_anndata.in
```

Comment:
Do we still need this one? Based on the README, we are only documenting the `scvi` one, so maybe we can remove the `AnnData` only one? Or if you're going to keep it, let's add it to the list in the README.

Reply:
I wanted to keep the separate `requirements_anndata` files, even though we may not be using them quite yet. I wasn't sure whether we really wanted a separate image at this point, but I thought having the separate requirements file was still worthwhile. I have an idea (in a separate branch) for how we may use it in combining all the dockerfiles into one that I will test once this goes in.

Reply:
Okay just wanted to check that it wasn't missed when you were building things.